app/main.py

```diff
@@ -1,5 +1,71 @@
+from typing import List, Optional
+
+
+class BaseRobot:
+    def __init__(self,
+                 name: str,
+                 weight: int,
+                 coords: Optional[List[int]] = None
+                 ) -> None:
+        self.name = name
+        self.weight = weight
+        self.coords = coords if coords is not None else [0, 0]
```

Comment:
You could simplify this line by using 
```suggestion
        self.coords = coords if coords else [0, 0]
```
or 
```suggestion
        self.coords = coords or [0, 0]
```